packages/cardpay-subgraph/schema.graphql

```diff
@@ -441,3 +442,13 @@ type TokenPair @entity {
   token1: Token!
   swaps: [TokenSwap]!             @derivedFrom(field: "tokenPair")
 }
+
+type RewardProgramRegistrationPayment @entity {
+  id: ID!
+  admin: Account!
+  transaction: Transaction!
+  createdAt: BigInt!
+  prepaidCardPayment: PrepaidCardPayment
```

Comment:
I have a feeling if I re-index from genesis block. this field `prepaidCardPayment` and `paidWith` will NOT be null so I can add the BANG (!) operator to require them. Let's check this.

Reply:
I think it is redundant to have _both_ prepaidCardPayment and PrepaidCard, since you can get to the PrepaidCard edge via the PrepaidCardPayment. But yes definitely these should use `!` operator.